sklearn/cross_validation.py

```diff
@@ -13,27 +12,29 @@
 from __future__ import division
 
 import warnings
-from itertools import chain, combinations
-from math import ceil, floor, factorial
 import numbers
-import time
+from itertools import combinations
 from abc import ABCMeta, abstractmethod
 
 import numpy as np
 import scipy.sparse as sp
+import sklearn.model_selection as mod_sel
```

Comment:
Thanks for the advice! (BTW if you can spare a few minutes, would you be able to review this? This has been around for a long time ;( )